rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -264,6 +264,7 @@ def __init__(
         index_label_id_mapping: Optional[Dict[int, Text]] = None,
         index_tag_id_mapping: Optional[Dict[int, Text]] = None,
         model: Optional[RasaModel] = None,
+        response_text_key_dict: Optional[Dict[Text, Any]] = None,
```

Comment:
Let's not add this to the `DIETClassifier` as it is specific to the `ResponseSelector`. Let's keep it only in the `ResponseSelector`, we can overwrite the persist and load in the sub-class.